source/coffee/image-picker.coffee

```diff
@@ -148,6 +148,9 @@ class ImagePickerOption
     else
       @option.text()
 
+  label_location: () ->
+    @option.text()
```

Comment:
I don't think it's good practice to read this from the `text` of the element,  it's probably better to read as data `@option.data("label-location")`